### ISSUE TITLE
Gradient might be doubled causing doubled learning rate

### ISSUE THREAD

**Issue by bobcao3** [taichi-dev/taichi-nerfs#75]
From https://github.com/taichi-dev/taichi/issues/8101

We might want to fix this to make it hyper-parameter wise compatible with torch solutions
